docs/user_guide.rst

```diff
@@ -178,6 +178,10 @@ or credentials. If you want to ignore specific parameters, specify them with ``i
     >>> session.get('http://httpbin.org/get', params={'auth-token': '2F63E5DF4F44'})
     >>> session.get('http://httpbin.org/get', params={'auth-token': 'D9FAEB3449D3'})
 
+In addition to allowing the cache to ignore these parameters when fetching cached results, these
```

Comment:
Could you also add a note that `ignore_parameters` applies to headers as well?

Reply:
Note added in commit 6465c8b